cmd/device-worker/main.go

```diff
@@ -29,6 +29,7 @@ import (
 )
 
 var yggdDispatchSocketAddr string
+var GracefulRebootChannel chan string
```

Comment:
Can the scope of it can be reduced to the main function or at least reduce the visibility?
Also, the type implies there is importance for the value. It seems we use it as a "light' state-machine.
Perhaps it will be simpler to have two channels, one for initiating the shutdown and one for terminating the device.